project/Dependencies.scala

```diff
@@ -4,21 +4,21 @@ import sbt.Keys._
 object Dependencies {
 
   // Version numbers
-  val PlayVersion = "2.5.13"
-  val AkkaVersion = "2.4.17"
+  val PlayVersion = "2.6.0-M3"
+  val AkkaVersion = "2.5.1"
   val ScalaVersion = "2.11.11"
-  val AkkaPersistenceCassandraVersion = "0.26"
-  val ScalaTestVersion = "3.0.1"
-  val JacksonVersion = "2.7.8"
-  val CassandraAllVersion = "3.0.9"
-  val GuavaVersion = "19.0"
+  val AkkaPersistenceCassandraVersion = "0.52"
+  val ScalaTestVersion = "3.0.3"
+  val JacksonVersion = "2.8.7"
+  val CassandraAllVersion = "3.8"
+  val GuavaVersion = "21.0"
   val MavenVersion = "3.3.9"
-  val NettyVersion = "4.0.42.Final"
-  val KafkaVersion = "0.10.0.1"
-  val AkkaStreamKafkaVersion = "0.13"
+  val NettyVersion = "4.1.8.Final"
+  val KafkaVersion = "0.10.2.0"
+  val AkkaStreamKafkaVersion = "0.15"
```

Comment:
They have since released 0.16 https://github.com/akka/reactive-kafka/milestone/16?closed=1

Though it isn't clear to me whether any of the fixes affect Lagom.